src/mixins/behaviors.js

```diff
@@ -0,0 +1,121 @@
+import _                        from 'underscore';
+import _getValue                from '../utils/_getValue';
+import getOption                from '../utils/getOption';
+import { _triggerMethod }       from '../trigger-method';
+import normalizeUIKeys          from '../utils/normalizeUIKeys';
+import BehaviorTriggersBuilder  from '../utils/behavior-trigger-builder';
+import parseBehaviors           from '../utils/parseBehaviors';
+
+// Borrow event splitter from Backbone
+var delegateEventSplitter = /^(\S+)\s*(.*)$/;
+
+function getBehaviorsUI(behavior) {
+  return behavior._uiBindings || behavior.ui;
+}
+
+export default {
+  _initBehaviors: function() {
+    var behaviors = _getValue(this.getOption('behaviors'), this);
+    var instanciatedBehaviors = {};
+
+    // Behaviors defined on a view can be a flat object literal
+    // or it can be a function that returns an object.
+    if (_.isObject(this.behaviors)) {
+      instanciatedBehaviors = parseBehaviors(this, behaviors);
+    }
+
+    this._behaviors = instanciatedBehaviors;
+    return behaviors;
+  },
+
+  _getBehaviorEvents: function() {
+    return _.result(this, 'behaviorEvents', {});
+  },
+
+  _getBehaviorTriggers: function() {
+    return _.result(this, 'behaviorTriggers', {});
+  },
+
+  behaviorTriggers: function(unusedBehaviorTriggers) {
+    var triggerBuilder = new BehaviorTriggersBuilder(this, this._behaviors);
+    return triggerBuilder.buildBehaviorTriggers();
+  },
+
+  behaviorEvents: function(unusedBehaviorEvents) {
```

Comment:
I think you could just rename this function `_getBehaviorEvents` also fortunately this `unusedBehaviorEvents` and the "unused" above this can go away